### DISCUSSION TITLE
How to Open Modal instead of login page

### DISCUSSION THREAD

**Discussion by abhinav-goel-11** [nextauthjs/next-auth#9668] [Help]
instead of a new login page i want to create a modal for signup and login how can i use my custom modal when signIn function is invoked  please tell🙏🙏🙏

**Comment by agoyal1999** [comment]
Why d'ont you just create a route f or login page but show a modal on UI for users to input their credentials? As using nextAuth you can only set a route for login but not a component. I would have done it like that.
set this in your authOptions:
```
 pages: {
    signIn: '/login',
  },
```

and create a /login/page.tsx which shows a modal.
